services.go

```diff
@@ -878,22 +882,9 @@ type SlackApplication struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/integrations.html#gitlab-for-slack-app
 type SlackApplicationProperties struct {
+	BranchesToBeNotified      string `json:"branches_to_be_notified"`
 	Channel                   string `json:"channel"`
 	NotifyOnlyBrokenPipelines bool   `json:"notify_only_broken_pipelines"`
-	BranchesToBeNotified      string `json:"branches_to_be_notified"`
-	AlertEvents               bool   `json:"alert_events"`
-	IssuesEvents              bool   `json:"issues_events"`
-	ConfidentialIssuesEvents  bool   `json:"confidential_issues_events"`
-	MergeRequestsEvents       bool   `json:"merge_requests_events"`
-	NoteEvents                bool   `json:"note_events"`
-	ConfidentialNoteEvents    bool   `json:"confidential_note_events"`
-	DeploymentEvents          bool   `json:"deployment_events"`
-	IncidentsEvents           bool   `json:"incidents_events"`
-	PipelineEvents            bool   `json:"pipeline_events"`
-	PushEvents                bool   `json:"push_events"`
-	TagPushEvents             bool   `json:"tag_push_events"`
-	VulnerabilityEvents       bool   `json:"vulnerability_events"`
-	WikiPageEvents            bool   `json:"wiki_page_events"`
 
```

Comment:
this should include this as well (sorry for the formatting from the cut and paste):
```
        LabelsToBeNotified         string `json:"labels_to_be_notified,omitempty"`
	LabelsToBeNotifiedBehavior string `json:"labels_to_be_notified_behavior,omitempty"`
	PushChannel                string `json:"push_channel,omitempty"`
	IssueChannel               string `json:"issue_channel,omitempty"`
	ConfidentialIssueChannel   string `json:"confidential_issue_channel,omitempty"`
	MergeRequestChannel        string `json:"merge_request_channel,omitempty"`
	NoteChannel                string `json:"note_channel,omitempty"`
	ConfidentialNoteChannel    string `json:"confidential_note_channel,omitempty"`
	TagPushChannel             string `json:"tag_push_channel,omitempty"`
	PipelineChannel            string `json:"pipeline_channel,omitempty"`
	WikiPageChannel            string `json:"wiki_page_channel,omitempty"`
	DeploymentChannel          string `json:"deployment_channel,omitempty"`
	IncidentChannel            string `json:"incident_channel,omitempty"`
	VulnerabilityChannel       string `json:"vulnerability_channel,omitempty"`
	AlertChannel               string `json:"alert_channel,omitempty"`
```